packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -37,8 +37,10 @@ export interface ContextSelectorProps extends Omit<ToggleMenuBaseProps, 'menuApp
   isFullHeight?: boolean;
   /** Labels the Context Selector for Screen Readers */
   screenReaderLabel?: string;
-  /** Text that appears in the Context Selector Toggle */
-  toggleText?: string;
+  /** Component or string that will be used in the Context Selector.
+   * This prop is placed in a button, and cannot contain any interactive elements
+   */
+  toggleText?: React.ReactNode | string;
```

Comment:
You do not need the "string" here.  ReactNode includes string type.

Reply:
Gotcha, I have updated the PR with the changes 👀